packages/app-builder/src/components/Analytics/Decisions.tsx

```diff
@@ -0,0 +1,207 @@
+      <div className="flex items-center gap-2">
+        <span className="text-s">Count:</span>
+        <div className="flex gap-1">
+          <ButtonV2
+            variant="secondary"
+            onClick={() => {
+              setPercentage(true);
+              setDecisionsState(
+                new Map([
+                  ['decline', true],
+                  ['blockAndReview', true],
+                  ['review', true],
+                  ['approve', true],
+                ]),
+              );
+            }}
+            className={percentage ? 'bg-purple-98 border-purple-65 text-purple-65' : ''}
+          >
+            %
+          </ButtonV2>
+          <ButtonV2
+            variant="secondary"
+            onClick={() => setPercentage(false)}
+            className={!percentage ? 'bg-purple-98 border-purple-65 text-purple-65' : ''}
+          >
+            #
+          </ButtonV2>
+        </div>
+      </div>
+      <div className="relative flex-1 w-full">
+        <ResponsiveBar
+          data={
+            percentage
+              ? (currentDataGroup?.data.ratio ?? [])
+              : (currentDataGroup?.data.absolute ?? [])
+          }
+          indexBy="date"
+          enableLabel={false}
+          keys={
+            percentage
+              ? ['decline', 'blockAndReview', 'review', 'approve']
+              : Array.from(decisionsState)
+                  .filter(([_, value]) => value)
+                  .map(([key]) => key)
+          }
+          padding={0.5}
+          margin={{ top: 80, right: 130, bottom: 50, left: 60 }}
+          colors={getBarColors}
+          defs={[
+            {
+              id: 'compareOpacity',
+              type: 'linearGradient',
+              colors: [
+                { offset: 0, color: 'inherit', opacity: 0.5 },
+                { offset: 100, color: 'inherit', opacity: 0.5 },
+              ],
+            },
+          ]}
+          fill={[
+            {
+              match: (n) => n.data.data['rangeId'] === 'compare',
+              id: 'compareOpacity',
+            },
+          ]}
+          axisLeft={{ legend: 'outcome (indexBy)', legendOffset: -70 }}
+          axisBottom={{
+            tickValues: currentDataGroup?.gridXValues,
+            format: (value: string) => {
+              // Convert the ISO string to a Date object and format it
+              const date = new Date(value);
+              return date.toLocaleDateString('fr-FR', {
+                month: 'short',
+                day: 'numeric',
+              });
+            },
+          }}
+          //   markers={currentDataGroup?.scenarioVersionsXMarkers}
+        />
+        <div className="relative bottom-2 right-2 flex gap-2">
+          <ButtonV2
+            variant="secondary"
+            mode="normal"
+            onClick={() => setGroupDate('daily')}
+            className={groupDate === 'daily' ? 'bg-purple-98 border-purple-65 text-purple-65' : ''}
```

Comment:
It's ok for now but maybe we should see with the designer if we eventually need a "selected/highlighted" version of the button...